recipe/meta.yaml

```diff
@@ -12,24 +12,24 @@ source:
 
 build:
   number: 0
-  script: {{ PYTHON }} -m pip install . --no-deps -vv
-  skip: true  # [py<35]
+  script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation -vv
+  skip: true  # [py<37 or s390x]
 
 requirements:
   host:
     - python
     - pip
+    - setuptools
+    - wheel
+    - flit-core 3.8.0
   run:
     - python
-    - numpy
-    - scipy >=1.0.0
-    - matplotlib-base
+    - numpy >=1.17.0
+    - scipy >=1.5.0
     - pandas
```

Comment:
I think pandas can be removed from the runtime dependencies. See https://github.com/search?q=repo%3AQuantEcon%2FQuantEcon.py%20pandas&type=code. It seems only used in the tests.

Reply:
Removed!